Add my Profile

kemiwilliam opened the pull request:
![kemiwilliam@PixGit](https://user-images.githubusercontent.com/18627431/55676584-9010b780-58cf-11e9-9291-45028bc6dd0a.PNG)

e0works commented:
You are targeting the wrong repo for this submission.

Ask your friends on Slack for assistance.

Yigaue committed:
create the classes

Yigaue committed:
create the index page for the blog

Yigaue committed:
did some styling for the blog

Yigaue committed:
create the admin/blogmanagers system

Yigaue committed:
creatte blog databse - this serve as a dummy/test database

Yigaue committed:
create blog route

Yigaue committed:
Done with BE, over to FE

Yigaue committed:
Done

Yigaue committed:
Merge branch 'staging' into master

Yigaue committed:
Merge pull request #1193 from Yigaue/master

Addition of single blog page -Backend

codypharm committed:
database created

codypharm committed:
database connection

Yigaue committed:
Merge pull request #1195 from iamwilliam19/staging

database created

iamadeyii committed:
Revert "Addition of single blog page -Backend"

iamadeyii committed:
Merge pull request #1196 from hnginternship5/revert-1193-master

Revert "Addition of single blog page -Backend"

abisalde committed:
page 404

OviePaulEJ committed:
Merge pull request #1 from hnginternship5/staging

Staging

billthebuilder committed:
Merge pull request #1197 from abisalde/staging

The new 404 page

Yigaue committed:
Routing of HNG Blog and FE

Yigaue committed:
Merge branch 'master' of https://github.com/Yigaue/hng.tech
merge conflict resolved

Yigaue committed:
Merge pull request #1198 from Yigaue/master

Routing for HNGblog and Some FE Work - No conflicts both with master and staging branch

damilare1947 committed:
Merge remote-tracking branch 'upstream/staging' into staging

Yigaue committed:
Merge pull request #1199 from damilare1947/staging

Staging

abisalde committed:
error page css modification

buikedim committed:
added products page

buikedim committed:
modified products page

Yigaue committed:
Merge pull request #1201 from buikedim/staging

created products page. the modified products page is the latest commit.

Yigaue committed:
Merge pull request #1200 from abisalde/staging

error page css modification

abisalde committed:
css correction

Yigaue committed:
Merge pull request #1202 from abisalde/staging

css correction

saucecodee committed:
Fe view post

Yigaue committed:
Merge pull request #1203 from saucecodee/staging

Fe view post page

saucecodee committed:
Fe view post

OviePaulEJ committed:
update join us page

OviePaulEJ committed:
Merge branch 'staging' of https://github.com/OviePaulEJ/hng.tech into staging

codecole committed:
Add Contact.php

This is the BE for the contact form for The HNG internship website for Dragonblades_hng

OviePaulEJ committed:
Merge branch 'staging' into staging

Yigaue committed:
integrated the databases

Yigaue committed:
Merge pull request #1209 from Yigaue/master

integrated the databases

damilare1947 committed:
removed the patterns in some of the text field

Yigaue committed:
Merge pull request #1210 from damilare1947/staging

removed the patterns in some of the text field

Yigaue committed:
Merge pull request #1205 from OviePaulEJ/staging

Adding Sign Up Page

damilare1947 committed:
did little changes in the input text

Yigaue committed:
Merge pull request #1211 from damilare1947/staging

did little changes in the input text

Yigaue committed:
Merge pull request #1207 from codecole/master

Add the contact form BE code

Yigaue committed:
Merge pull request #1204 from saucecodee/staging

Fe view post

OviePaulEJ committed:
Merge pull request #2 from hnginternship5/staging

Update Local Staging

damilare1947 committed:
included alert after submitting in the contact.php page

damilare1947 committed:
removed data-target

damilare1947 committed:
added some styling to app.css, completed the interns.php page and linked the contact button to the contact page on footer.php

buikedim committed:
added final features to product page

buikedim committed:
modified products page

buikedim committed:
modified product page

abisalde committed:
modification

OviePaulEJ committed:
Merge pull request #1216 from abisalde/staging

modification

OviePaulEJ committed:
Merge pull request #1215 from buikedim/staging

modified product page

saucecodee committed:
Add recommended post section

Yigaue committed:
admin page styling

Yigaue committed:
Merge branch 'staging' into mybranch1

Yigaue committed:
Merge pull request #1219 from Yigaue/mybranch1

admin page styling

Yigaue committed:
Merge branch 'staging' into staging

Yigaue committed:
Merge pull request #1217 from saucecodee/staging

Added recommended post section

damilare1947 committed:
changed the style form-group to form-contact

Yigaue committed:
added some styling on the blog

Yigaue committed:
Merge pull request #1221 from Yigaue/master

added some styling on the blog

damilare1947 committed:
Merge remote-tracking branch 'upstream/staging' into staging

OviePaulEJ committed:
Merge pull request #1212 from damilare1947/staging

Staging

saucecodee committed:
Add moble view and other fixes

Yigaue committed:
Merge pull request #1223 from saucecodee/staging

Added moble view and other fixes

damilare1947 committed:
did a ltittle fixe on the input text for full name

codecole committed:
Add the BE contact page HNG

Yigaue committed:
Merge pull request #1225 from codecole/staging

Add the BE contact page HNG

Yigaue committed:
Merge pull request #1224 from damilare1947/staging

did a ltittle fixe on the input text for full name

OviePaulEJ committed:
Merge pull request #3 from hnginternship5/staging

Update Staging

OviePaulEJ committed:
update join-now page

OviePaulEJ committed:
Revert "Update Staging"

OviePaulEJ committed:
Merge pull request #4 from OviePaulEJ/revert-3-staging

Revert 3 staging

OviePaulEJ committed:
Merge pull request #5 from OviePaulEJ/revert-3-staging

Revert "Update Staging"

OviePaulEJ committed:
Merge branch 'staging' of https://github.com/OviePaulEJ/hng.tech into staging

abisalde committed:
Meet Our Interns Page

OviePaulEJ committed:
add styling to join-now page and update backend ids

OviePaulEJ committed:
Merge pull request #1227 from OviePaulEJ/staging

Adding join-now page

OviePaulEJ committed:
Merge branch 'staging' into staging

OviePaulEJ committed:
Merge pull request #1226 from abisalde/staging

Meet Our Interns Page

Yigaue committed:
add query to fetch author, update database, did some styling

Yigaue committed:
Merge branch 'staging' into master

Yigaue committed:
Merge pull request #1228 from Yigaue/master

add query to fetch author, update database, did some styling

Yigaue committed:
minor changes

Yigaue committed:
Merge pull request #1229 from Yigaue/master

minor changes

saucecodee committed:
Fixed page

Yigaue committed:
Merge pull request #1230 from saucecodee/staging

Fixed page

Yigaue committed:
improve navigation

Yigaue committed:
Merge pull request #1231 from Yigaue/master

improve navigation

Yigaue committed:
corrected navigation

Yigaue committed:
adjust return to blog

Yigaue committed:
Merge pull request #1232 from Yigaue/master

corrected navigation

Yigaue committed:
correcetd

Yigaue committed:
well correctd

Yigaue committed:
Merge pull request #1233 from Yigaue/master

Well corrected

damilare1947 committed:
updated my contact.php

OviePaulEJ committed:
add drop down options

Yigaue committed:
Merge pull request #1234 from damilare1947/staging

updated my contact.php. i didn't touch the app.css file

OviePaulEJ committed:
Merge pull request #6 from hnginternship5/staging

Updateed staging from Sauce Code

OviePaulEJ committed:
Merge https://github.com/OviePaulEJ/hng.tech into staging

OviePaulEJ committed:
Merge branch 'staging' of https://github.com/OviePaulEJ/hng.tech into staging

OviePaulEJ committed:
Merge pull request #1235 from OviePaulEJ/staging

Drop down update to Join Us page

OviePaulEJ committed:
update join-us form CSS

OviePaulEJ committed:
Merge pull request #1236 from OviePaulEJ/staging

update join-us form CSS

abisalde committed:
Merge https://github.com/abisalde/hng.tech into staging

abisalde committed:
changes for meet intern

abisalde committed:
modification

buikedim committed:
products page

abisalde committed:
changes to meet our intern page

Yigaue committed:
FE work on the admin dashboard

OviePaulEJ committed:
Merge pull request #1239 from buikedim/staging

products page

abisalde committed:
all

damilare1947 committed:
changed form action to process on contact.php page

codecole committed:
add BE for contact page.

Yigaue committed:
Merge pull request #1244 from codecole/staging

add BE for contact page.

OviePaulEJ committed:
Merge pull request #1243 from damilare1947/staging

changed form action to process on contact.php page

Yigaue committed:
admin dashboard and post category setup

Yigaue committed:
Merge pull request #1245 from Yigaue/master

admin dashboard and category setup

damilare1947 committed:
Update contact.php

Yigaue committed:
Merge pull request #1246 from damilare1947/staging

Update contact.php

abisalde committed:
modification for css

abisalde committed:
Merge pull request #1 from hnginternship5/staging

Updated CSS from Meet Our Interns and 404

abisalde committed:
Merge branch 'staging' of https://github.com/abisalde/hng.tech into staging

OviePaulEJ committed:
Merge pull request #1247 from abisalde/staging

CSS Modification for Meet Our Inetrns and 404

Yigaue committed:
fixed bugs on product page

Yigaue committed:
Merge pull request #1248 from Yigaue/master

fixed bugs on product page

OviePaulEJ committed:
Merge pull request #7 from hnginternship5/staging

Staging Update 7pm May 7th

OviePaulEJ committed:
add verification to join-us page

OviePaulEJ committed:
Merge branch 'staging' of https://github.com/OviePaulEJ/hng.tech into staging

Yigaue committed:
update database

Yigaue committed:
Merge pull request #1249 from Yigaue/master

update database

OviePaulEJ committed:
add validation to join-us form

OviePaulEJ committed:
Merge pull request #8 from hnginternship5/staging

Update staging 11:25PM May 7

OviePaulEJ committed:
Merge branch 'staging' of https://github.com/OviePaulEJ/hng.tech into staging

OviePaulEJ committed:
Merge pull request #1250 from OviePaulEJ/staging

Update validation on Join-now form

OviePaulEJ committed:
link sumbit data to database and add confirmation message

OviePaulEJ committed:
Merge pull request #1251 from OviePaulEJ/staging

link sumbit data to database and add confirmation message

Yigaue committed:
automatically fetch interns details

Yigaue committed:
Merge pull request #1252 from Yigaue/master

automatically fetch interns details

Yigaue committed:
create function to filter the files fetched into meet-our-intern page

Yigaue committed:
Merge pull request #1253 from Yigaue/master

create function to filter the files fetched into meet-our-intern page

Yigaue committed:
updated database

Yigaue committed:
Merge pull request #1254 from Yigaue/master

updated database

Commit:
added akwa-ibom logo

OviePaulEJ committed:
Merge pull request #1255 from seyicole/staging

added akwa-ibom logo

codecole committed:
Delete process.php

codecole committed:
add: Mail Configuration done

Mail configuration done with membersdrive API

Yigaue committed:
Merge pull request #1256 from codecole/staging

Staging

buikedim committed:
modified products page

Commit:
added blog listing page

Commit:
added active nav

rapulu committed:
resolve merged confict

emkayDauda committed:
Add emkayDauda.php to staging

saucecodee committed:
Fix input alignment

Yigaue committed:
Merge pull request #1261 from saucecodee/staging

Fix input alignment

OviePaulEJ committed:
Merge pull request #1258 from buikedim/staging

modified products page

Yigaue committed:
dummy json file for interns finalist page

Yigaue committed:
Merge pull request #1262 from Yigaue/master

dummy json file for interns finalist page

OviePaulEJ committed:
Merge branch 'staging' into staging

OviePaulEJ committed:
Merge pull request #1259 from seyicole/staging

Staging

Yigaue committed:
fixed bug on fetching post author

Yigaue committed:
Merge pull request #1264 from Yigaue/master

fixed bug on fetching post author

rapulu committed:
added env settings

rapulu committed:
Merge branch 'staging' of github.com:hnginternship5/hng.tech into staging

rapulu committed:
Merge branch 'staging' into env

Yigaue committed:
Merge pull request #1260 from emkayDauda/staging

Add emkayDauda.php to staging

emmanuelnwankwo committed:
Profile updated

emmanuelnwankwo committed:
Profile updated

tomisinlalude committed:
Create the HNG Finalists Page and integrate to the repo

tomisinlalude committed:
Merge branch 'staging' into staging

tomisinlalude committed:
Merge pull request #1266 from OluwatomisinLalude/staging

Create the HNG Finalists Page and integrate to the repo

tomisinlalude committed:
Change route of "See More" button

tomisinlalude committed:
Change route in "See More" button

rapulu committed:
added hng5 finalist route

rapulu committed:
removed extra cards

rapulu committed:
finalist script adde

tomisinlalude committed:
Modify hng finalist page

tomisinlalude committed:
Merge branch 'staging' of https://github.com/hnginternship5/hng.tech into staging

rapulu committed:
fetch functions

rapulu committed:
spreadsheet api setup

tomisinlalude committed:
Modify finalists page

tomisinlalude committed:
Fix conflicts

tomisinlalude committed:
Modify finalists page

tomisinlalude committed:
Merge pull request #1267 from OluwatomisinLalude/staging

Modify finalists page

rapulu committed:
removed card

rapulu committed:
added digital marketers

rapulu committed:
commented tracks

rapulu committed:
Merge pull request #1268 from rapulu/staging

Code refactoring

OviePaulEJ committed:
Merge pull request #9 from hnginternship5/staging

Staging-hng5-finalist

OviePaulEJ committed:
remove card background and fix card flip

OviePaulEJ committed:
Merge pull request #1269 from OviePaulEJ/staging

Staging

OviePaulEJ committed:
update the card flip on DM section

OviePaulEJ committed:
Merge pull request #1270 from OviePaulEJ/staging

update the card flip on DM section

rapulu committed:
finalist page completed

buikedim committed:
modified products page

rapulu committed:
resolved merge conflict

rapulu committed:
console commented out

rapulu committed:
Merge pull request #1272 from rapulu/staging

final list page fixed

rapulu committed:
fixed finalist route in interns

rapulu committed:
Merge pull request #1273 from rapulu/staging

fixed finalist route on interns

OviePaulEJ committed:
Merge pull request #1271 from buikedim/staging

modified products page

OviePaulEJ committed:
Merge pull request #10 from hnginternship5/staging

Staging

OviePaulEJ committed:
remove blog

OviePaulEJ committed:
remove read intern stories button from join now submission

rapulu committed:
added storage folder

OviePaulEJ committed:
fix see more section on interns page

OviePaulEJ committed:
fix button and text on Meet our Finalist section

OviePaulEJ committed:
resolved conflicts

OviePaulEJ committed:
make products link to external pages

OviePaulEJ committed:
Merge pull request #1275 from OviePaulEJ/staging

Fixed highlited bugs on the page

OviePaulEJ committed:
fix hng5-finalists page container

OviePaulEJ committed:
Merge pull request #1276 from OviePaulEJ/staging

fix hng5-finalists page container

rapulu committed:
finalist csv setup

rapulu committed:
resolved conflict on finalist page

rapulu committed:
Merge pull request #1277 from rapulu/staging

csv implemented for finalist page

OviePaulEJ committed:
Merge pull request #11 from hnginternship5/staging

Staging

OviePaulEJ committed:
update lucid logo on product page

OviePaulEJ committed:
Merge branch 'staging' of https://github.com/OviePaulEJ/hng.tech into staging

OviePaulEJ committed:
fix card back flip and remove white h1s

OviePaulEJ committed:
Merge pull request #1278 from OviePaulEJ/staging

Updates on product page and fix on finalist card flip

OviePaulEJ committed:
add emails to cards

OviePaulEJ committed:
Merge pull request #1279 from OviePaulEJ/staging

add emails to cards

rapulu committed:
web developers email updated

rapulu committed:
Merge pull request #1280 from rapulu/staging

web developers email updated

OviePaulEJ committed:
Merge pull request #12 from hnginternship5/staging

Staging

OviePaulEJ committed:
fix card margin errors

OviePaulEJ committed:
Merge branch 'staging' of https://github.com/OviePaulEJ/hng.tech into staging

OviePaulEJ committed:
will remove hng coins payment and include verifi logo on supporters section

OviePaulEJ committed:
update text on Apoe4 description

OviePaulEJ committed:
update header on what is the internship page

OviePaulEJ committed:
Update products.php

OviePaulEJ committed:
Merge pull request #1281 from OviePaulEJ/staging

Bug fixes

OviePaulEJ committed:
fix car flip on Mozilla browser

OviePaulEJ committed:
Merge branch 'staging' of https://github.com/OviePaulEJ/hng.tech into staging

OviePaulEJ committed:
Merge pull request #1282 from OviePaulEJ/staging

Fix card flip on Mozilla

rapulu committed:
inters data updated

rapulu committed:
Merge pull request #1283 from rapulu/staging

interns data updated

rapulu committed:
updated interns data

rapulu committed:
Merge pull request #1285 from rapulu/staging

updated interns data

Commit:
fixed intern responsiveness

Commit:
Update app.css

rapulu committed:
Merge pull request #1286 from clarencepenz/staging

fixed responsiveness

chibuikeIg committed:
just a commit

chibuikeIg committed:
Merge branch 'master' of https://github.com/hnginternship5/hng.tech into staging
updated branch

chibuikeIg committed:
added interns details and current work place

rapulu committed:
Merge pull request #1287 from chibuikeIg/staging

Staging
